components/automate-ui/src/app/modules/infra-proxy/org-details/org-details.component.html

```diff
@@ -0,0 +1,124 @@
+              Admin User is required.
+            </chef-error>
+          </chef-form-field>
+          <chef-form-field>
+            <label>
+            <span class="label">Admin Key<span aria-hidden="true">*</span></span>
+            <textarea rows="27" cols="10" chefInput placeholder="-----BEGIN RSA PRIVATE KEY -----"
+              formControlName="admin_key" data-cy="update-org-admin-key"></textarea>   
+            </label>
+            <chef-error
+              *ngIf="(updateOrgForm.get('admin_key').hasError('required') || updateOrgForm.get('admin_key').hasError('pattern')) && updateOrgForm.get('admin_key').dirty">
+              Admin Key is required.
+            </chef-error>
+          </chef-form-field>
+          <chef-form-field>
+            <div id="button-bar">
+              <chef-button [disabled]="isLoading || !updateOrgForm.valid || !updateOrgForm.dirty" primary
+              inline (click)="saveOrg()">
+                <chef-loading-spinner *ngIf="saveInProgress"></chef-loading-spinner>
+                <span *ngIf="saveInProgress">Saving...</span>
+                <span *ngIf="!saveInProgress">Save</span>
+              </chef-button>
+              <span id="saved-note" *ngIf="saveSuccessful && !updateOrgForm.dirty">All changes
+              saved.</span>
+            </div>
+          </chef-form-field>
+        </form>
+      </section>
+      <section class="page-body" *ngIf="tabValue === 'cookbooks'">
+        <chef-loading-spinner *ngIf="cookbooksListLoading" size="50"></chef-loading-spinner>
+        <ng-container  *ngIf="!cookbooksListLoading">
+          <chef-table>
+            <chef-thead>
+              <chef-tr>
+                <chef-th>Cookbook Name</chef-th>
+                <chef-th>Cookbook Version</chef-th>
+              </chef-tr>
+            </chef-thead>
+            <chef-tbody>
+              <chef-tr *ngFor="let cookbook of cookbooks">
+                <chef-td>{{ cookbook.name }}</chef-td>
+                <chef-td>{{ cookbook.version }}</chef-td>
+              </chef-tr>
+            </chef-tbody>
+          </chef-table>
+        </ng-container>
+      </section>
+      <section class="page-body" *ngIf="tabValue === 'roles'">
+        <chef-loading-spinner *ngIf="rolesListLoading" size="50"></chef-loading-spinner>
+        <ng-container *ngIf="!rolesListLoading">
+          <chef-table>
+            <chef-thead>
+              <chef-tr>
+                <chef-th>Role Name</chef-th>
+                <chef-th>Description</chef-th>
+                <chef-th>Environments</chef-th>
+              </chef-tr>
+            </chef-thead>
+            <chef-tbody>
+              <chef-tr *ngFor="let role of roles">
+                <chef-td>
+                  <a [routerLink]="['/infrastructure','chef-servers', serverId, 'org', OrgId, 'roles', role.name]">{{ role.name }}</a>
```

Comment:
FYI - Make sure to take a closer look at this after you rebase master.  We made some updates to the chef-table components recently.  All the naming conventions are the same, so everything is _probably_ fine, but you'll want to double check just incase.

Reply:
I have already updated the changes as per master branch updations :)